.github/CODEOWNERS

```diff
@@ -0,0 +1,76 @@
+# Describes which individuals/teams are reponsible for which code in the
+# repository.
+#
+# See: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/.cargo                             @benesch
+/.config                            @benesch
+/.devcontainer                      @mjibson
+/.github                            @benesch
+/bin
+/ci                                 @benesch
+/doc/user                           @MaterializeInc/DevEx
+/doc/developer/reference/compute    @MaterializeInc/compute
+/doc/developer/reference/storage    @MaterializeInc/storage
+/ci                                 @MaterializeInc/qa
+/src/adapter                        @MaterializeInc/surfaces
+/src/alloc                          @benesch
+/src/audit-log                      @MaterializeInc/surfaces
+/src/avro                           @umanwizard
+/src/avro-derive                    @umanwizard
+/src/build-id                       @umanwizard
+/src/build-info                     @benesch
+/src/ccsr                           @benesch
+/src/cloud-resources                @MaterializeInc/cloud
+/src/clusterd                       @MaterializeInc/compute @MaterializeInc/storage
+/src/compute                        @MaterializeInc/compute
+/src/compute-client                 @MaterializeInc/compute
+/src/environmentd                   @MaterializeInc/surfaces
+/src/expr                           @MaterializeInc/compute
+/src/expr-test-util                 @MaterializeInc/compute
+/src/frontegg-auth                  @MaterializeInc/compute
```

Comment:
This doesn't seem right!

Reply:
Funny, that's what my attempt to script this came up with, too, and is the peril of the otherwise-useful approach of basically `git shortlog -n -s path`, but looking at the commits, all the compute-team contributions here are actually repo-spanning changes, and Matt and Andreas seem to have the only real changes here.  (Detecting this case is something I meant to do in the code for generating this I was resurrecting a while back.)

Reply:
Oh, right, I meant to assign that to surfaces (Matt).